go.mod

```diff
@@ -361,12 +367,7 @@ require (
 	sigs.k8s.io/yaml v1.4.0 // indirect
 )
 
-replace (
-	// until merged upstream: https://github.com/omissis/go-jsonschema/pull/264
-	github.com/atombender/go-jsonschema => github.com/nolag/go-jsonschema v0.16.0-rtinianov
```

Comment:
:fire:

Reply:
Yeah, I should have done that sooner.  I forgot to update core after I updated common.